src/libmongoc/tests/test-mongoc-cursor.c

```diff
@@ -702,22 +708,40 @@ test_kill_cursor_live (void)
 
    ASSERT_CMPINT (ctx.succeeded_count, ==, 1);
 
-   b = bson_new ();
-   cursor = _mongoc_cursor_find_new (
-      client, collection->ns, b, NULL, NULL, NULL, NULL);
-   /* override the typical priming, and immediately transition to an OPQUERY
-    * find cursor. */
-   cursor->impl.destroy (&cursor->impl);
-   _mongoc_cursor_impl_find_opquery_init (cursor, b);
+   if (test_framework_supports_legacy_opcodes ()) {
```

Comment:
Are these branches significantly different enough to warrant different tests with different skipping conditions?

Reply:
IMO no, since how the `getMore` operation is run is independent of what is being tested.

The intent of the test is to ensure killing a cursor works.

The `getMore` operation is used to verify the cursor no longer exists server side.